### PR TITLE
Assignment 3 and Final task CNN_Vorobeva M

### PR DESCRIPTION
**Pull request by Murwar**
@toshiks 
Задание 3.1: была реализована сверточная нейронная сеть. Точность на тренировочных данных дошла до 100%  за 36 epochs. Правда, это было один раз и больше не повторялось...
Loss: 0.041467, Train accuracy: 1.000000, val accuracy: 0.117188

Задание 3.2:  была имплементирована архитектура LeNet. В качестве своей модели была использована также архитектура LeNet c добавлением batchNormalization и Softmax. Точность на тестовой выборке составила ~83%, однако модель прошла всего 10 epoch. С моделью еще экспериментирую.

Подбор гиперпараметров: 
Best validation accuracy: 0.81, best hyperparams: Hyperparams(learning_rate=0.001, anneal_epochs=15, reg=1e-05)

### CODE REVIEW

**Comment by Murwar** [comment]
Реализовала модель VGG'11 и подобрала к ней параметры, точность стала лучше: Final test accuracy -  0.936

**Comment by Murwar** [comment]
@toshiks 
Final task_CNN: Был натренирован клаccификатор, который отличает хотдоги от не хотдогов. Реализована модель на основе resnet18. F1 score на validation set на данной модели доведен до значения 0.936. 

Во время выполнения задания узнала, что у colab есть лимит...
